frontend/app/controllers/Event.scala

```diff
@@ -167,6 +167,7 @@ trait Event extends Controller with ActivityTracking {
       trackAnon(EventActivity("eventThankYou", memberData, EventData(event), order.map(OrderData(_))))(request)
       event.service.wsMetrics.put("user-returned-to-thankyou-page", 1)
     }
+    event.service.wsMetrics.put("user-returned-to-thankyou-page-no-cookie", 1)
```

Comment:
Good call – done.

Also, looking at Google Analytics, for the thankyou page we're seeing 325 pageviews, 302 unique – so only 8% of these calls are duplicate, not too many.